etc/servo_gdb.py

```diff
@@ -7,19 +7,22 @@
 # option. This file may not be copied, modified, or distributed
 # except according to those terms.
 
-# A set of simple pretty printers for gdb to make debugging Servo a bit easier.
+"""
```

Comment:
To those unfamiliar with Python, this is a module level docstring, which seems appropriate considering the contents. Flake8 did not like the `#texthere` conventions for comments without a space inbetween the text and the `#` so this worked out